docs/user/document/distribution/tlp/label-usage.en.md

```diff
@@ -1,22 +1,22 @@
 # Label - Usage
 
-* The label should be set and must be valid. There are four color (or traffic light) values:
+The label should be set and be valid. There are four color (or traffic light) values:
```

Comment:
The double `be` stings ;-) Please remind me: Why did we remove the `must`?

Please remove "color or (" and just use "traffic light values" as this will change in the next version

Reply:
removed `color (or`.

The `must` is removed because the statement is derived from the _optional_ test 6.2.10.

Reply:
But the `must` is enforce via the schema, isn't it?

Reply:
Yes, as this is an enum with defined values it is. I added the `must` again.